src/Swashbuckle.AspNetCore.Swagger/DependencyInjection/SwaggerBuilderExtensions.cs

```diff
@@ -25,5 +38,37 @@ public static class SwaggerBuilderExtensions
 
             return app;
         }
+#if NETCOREAPP3_0
+        public static IEndpointRouteBuilder MapSwagger(
+            this IEndpointRouteBuilder endpoints,
+            string pattern,
```

Comment:
In place of the overload method below, could we just make the `pattern` property optional and default it to "/swagger/{documentName}/swagger.json". This way, the default value will be more visible via intellisense.